syntax = "proto2";

message g2s_player_enter_response {
	required uint64 player_id = 1;
	required uint32 result = 2;
}